thoth/messaging/__init__.py

```diff
@@ -26,5 +26,14 @@
 from .unresolved_package import UnresolvedPackageMessage
 from .message_factory import message_factory
 
+ALL_MESSAGES = [
+    HashMismatchMessage,
+    MissingVersionMessage,
+    MissingPackageMessage,
+    PackageReleaseMessage,
+    UnresolvedPackageMessage,
+]
+
+
 __name__ = "thoth-messaging"
 __version__ = "0.5.0"
```

Comment:
I think there is need to rebase also after @harshad16 PR: https://github.com/thoth-station/messaging/commit/cfd129f6a588eda1211058143440d5f0ba9f232d